go/genkit/genkit.go

```diff
@@ -23,6 +23,35 @@ import (
 	"github.com/firebase/genkit/go/core"
 )
 
+// Options are options to [Init].
+type Options struct {
+	DevAddr string
+	// If "-", do not start a FlowServer.
+	// Otherwise, start a FlowServer on the given address, or the
+	// default of ":3400" if empty.
+	FlowAddr string
```

Comment:
We probably want options to specify which flows we want to expose via the flows server. Perhaps this would work better as a strict?

`FlowsServer: {Addr:..., Flows:...}`

Reply:
done